utils/utils_test.go

```diff
@@ -23,13 +23,10 @@ func TestStringArrayIntersection(t *testing.T) {
 		"def",
 	}
 
-	if len(StringArrayIntersection(a, b)) != 0 {
-		t.Fatal("should be 0")
-	}
+	assert := assert.New(t)
```

Comment:
For consistency, we prefer to not use this approach, we pass the `t` parameter in each call.